src/Core/Models/View/SendFileView.cs

```diff
@@ -0,0 +1,25 @@
+using System.Dynamic;
+using Bit.Core.Models.Domain;
+
+namespace Bit.Core.Models.View
+{
+    public class SendFileView : View
+    {
+        public SendFileView() { }
+
+        public SendFileView(SendFile file)
+        {
+            Id = file.Id;
```

Comment:
would `file` parameter ever be `null`?

Reply:
As intended to be used, no. All `View` objects are created exclusively by a `DecryptAsync` call on their respective `Domain` objects, which pass themselves in as the parameter here. For example, the only reference to this constructor is in SendFile.cs and reads `new SendFileView(this)`

This behavior matches all the other `View`s as well. I'm totally fine with a guard clause returning `null`, but I think I'd want to write one on all the other `View` objects too